tracks/go/exercises/hamming/mentoring.md

```diff
@@ -41,9 +41,11 @@ The most common feedback revolves around:
 * Readability (formatting, comments, code structure)
 
 **Error and error message: correct instantiation and concise error message**
-* Does the error message describe the problem? In Go standard error the error message is all we have. So it should be concise describing the problem in a non-generical way.
-* Is the error formatted correctly? For example the error [should not be capitalized or have punctuation](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)
+* Is the error message empty or vague? In Go the returned error can provide context for where the error occurred and why, so it should concisely and specifically describe the problem.
+* Is the error formatted correctly? For example the error [should not be capitalized or have punctuation](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).
 * Did they use a combination of `errors.New` and `fmt.Sprintf`? Hint at `golint` and `fmt.Errorf`.
+* Did they use `fmt.Errorf` for a non-interpolated string? It may be worth suggesting they use `errors.New` in these cases to explicitly indicate that no interpolation will be occurring. Encouraging early exposure to the `errors` package might start benefitting them immediately, e.g., `errors.Wrap`.
+* Did they return `-1` or `0` in the error case? Returning the zero value may be more idiomatic, since the error indicates that the preceding value isn't that useful. There are test cases that specify `-1` should be returned, but because this value isn't checked by the tests, a student doing strict TDD might not discover this. Careful readers might. 
```

Comment:
```suggestion
* Did they return `-1` or `0` in the error case? Returning the zero value may be more idiomatic, since the error indicates that the preceding value isn't that useful. There are test cases that specify `-1` should be returned, but because this value isn't checked by the tests, a student doing strict TDD might not discover this; careful readers might. 
```